frb_dart/lib/src/helpers.dart

```diff
@@ -148,3 +148,15 @@ Duration wire2apiDuration(int ts) {
   }
   return Duration(microseconds: ts);
 }
+
+Uint8List wire2apiConcatenateBytes(List raw) {
+  var builder = BytesBuilder();
+  if (raw.isEmpty) {
+    return builder.toBytes();
+  }
+  final count = raw.length;
+  for (var i = 0; i < count; i++) {
```

Comment:
nit: can be replaced with for-each loop and ignore the empty check

Reply:
resolved in 8be4048bd8476aa23480a143e586ed694c75b28d